woocommerce/woocommerce-gutenberg-products-block
# Filter Products by Price: Block creation

Closes https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/786

This PR takes care of the block creation for the filter products by price block. It does not make it functional, that will come in future PRs.

**Screenshots**

If you have no products, you'll see a placeholder:

![Screenshot 2019-08-15 at 10 27 51](https://user-images.githubusercontent.com/90977/63086449-c928fa80-bf47-11e9-9baa-4ef8b0013676.png)

Otherwise you'll see the slider, which has several variations based on settings:

![Screenshot 2019-08-15 at 10 26 46](https://user-images.githubusercontent.com/90977/63086471-d7771680-bf47-11e9-85c9-d66abd3886a6.png)
![Screenshot 2019-08-15 at 10 26 57](https://user-images.githubusercontent.com/90977/63086474-dba33400-bf47-11e9-9195-49e41ae8af54.png)

The settings include:

![Screenshot 2019-08-15 at 10 24 48](https://user-images.githubusercontent.com/90977/63086492-e3fb6f00-bf47-11e9-9f92-04cd898b8afa.png)

**Testing instructions**

- Create a price slider block with no products on your store. Confirm placeholder is shown and buttons are functional.
- Create products.
- Create price slider block. Play around with the block settings and ensure the display updates.
- Save, ensure settings are preserved.
